src/series8/Income.java

```diff
@@ -0,0 +1,31 @@
+package series8;
+
+import java.util.Scanner;
+
+public class Income {
+	public static void main(String[] args)
+	{
+		int income_input;
+		
+		Scanner sc = new Scanner(System.in);
+		
+		income_input = sc.nextInt();
+		
+		if(income_input >= 100000)
+		{
+			System.out.println("You have a high income. Congrajulations!");
+		}
+		else if(income_input >= 50000)
+		{
+			System.out.println("You have a moderate income");
```

Comment:
Your indentation inside else if is too big. Generally its four spaces or two spaces.

```
if (xyz) {
  do something
}

if (xyz) {
    do something
}
```